src/media/mod.rs

```diff
@@ -265,6 +265,7 @@ impl Media {
                     network_time: dep.first_network_time(),
                     seq_range: dep.seq_range(),
                     contiguous: dep.contiguous,
+                    audio_start_of_talk_spurt: dep.start_of_talkspurt(),
```

Comment:
Add a check for `codec.spec().codec.is_audio()` here to avoid setting this to true when marker is used as end-of-video-frame.